projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-following.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * REST API endpoint for the Jetpack author recommendations block.
+ *
+ * @package automattic/jetpack
+ * @since $$next-version$$
+ */
+
+use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Status\Visitor;
+
+/**
+ * Class WPCOM_REST_API_V2_Endpoint_Following
+ */
+class WPCOM_REST_API_V2_Endpoint_Following extends WP_REST_Controller {
+	/**
+	 * Namespace prefix.
+	 *
+	 * @var string
+	 */
+	public $namespace = 'wpcom/v2';
+
+	/**
+	 * Endpoint base route.
+	 *
+	 * @var string
+	 */
+	public $rest_base = 'following';
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->wpcom_is_wpcom_only_endpoint    = true;
+		$this->wpcom_is_site_specific_endpoint = true;
```

Comment:
I think this needs to be false, or else it will require `/sites/SITE_ID` to be appended to the endpoint.